packages/xsnap/test/test-xs-perf.js

```diff
@@ -138,7 +139,9 @@ test('metering can be switched off / on at run-time', async t => {
   const opts = options(io);
   const vat = xsnap(opts);
   t.teardown(() => vat.terminate());
-  const { meterUsage: { compute: noUnMeteredCompute } } = await vat.evaluate(`
+  const {
+    meterUsage: { compute: noUnMeteredCompute },
```

Comment:
nit: Maybe `fullyMeteredCompute` to cancel the negatives. I want to love giving `Un` its own camel hump, and also `MoDem`, `TeleType`, and `TransCoder`.